superset/charts/schemas.py

```diff
@@ -165,6 +170,11 @@ class ChartEntityResponseSchema(Schema):
     slice_url = fields.String(description=slice_url_description)
     certified_by = fields.String(description=certified_by_description)
     certification_details = fields.String(description=certification_details_description)
+    is_managed_externally = fields.Boolean(
+        description=is_managed_externally_description
+    )
+    owners = fields.List(fields.Nested(UserSchema))
```

Comment:
a minor suggestion from recent fixing. the `owners` fields suggest that align with [Dashboard](https://github.com/apache/superset/pull/20826/files#) field to `owners = fields.List(fields.Dict())`.

Reply:
Hmm, I'm getting a 500 from the dashboard endpoint with that change
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/15073128/180833546-4e1524c9-f3fa-4705-8ab4-641a7c00e1d1.png">